### DISCUSSION TITLE
tree-select  does not support remote filtering ?

### DISCUSSION THREAD

**Discussion by WestFarmer** [element-plus/element-plus#15300] [Q&A]
multiple value, lazy load, remote filtering.

can any one share a example ?

**Comment by ryuhangyeong** [comment]
Is this the example you want?

[Here](https://element-plus.run/#eyJzcmMvQXBwLnZ1ZSI6Ijx0ZW1wbGF0ZT5cbiAgPGVsLXRyZWUtc2VsZWN0XG4gICAgdi1tb2RlbD1cInZhbHVlMlwiXG4gICAgbGF6eVxuICAgIG11bHRpcGxlXG4gICAgOmxvYWQ9XCJsb2FkXCJcbiAgICA6cHJvcHM9XCJwcm9wc1wiXG4gICAgOmNhY2hlLWRhdGE9XCJjYWNoZURhdGFcIlxuICAgICAgOmZpbHRlci1ub2RlLW1ldGhvZD1cImZpbHRlck5vZGVNZXRob2RcIlxuICAvPlxuPC90ZW1wbGF0ZT5cblxuPHNjcmlwdCBsYW5nPVwidHNcIiBzZXR1cD5cbmltcG9ydCB7IHJlZiB9IGZyb20gJ3Z1ZSdcblxuY29uc3QgdmFsdWUgPSByZWYoKVxuY29uc3QgdmFsdWUyID0gcmVmKDUpXG5cbmNvbnN0IGNhY2hlRGF0YSA9IFt7IHZhbHVlOiA1LCBsYWJlbDogJ2xhenkgbG9hZCBub2RlNScgfV1cblxuY29uc3QgcHJvcHMgPSB7XG4gIGxhYmVsOiAnbGFiZWwnLFxuICBjaGlsZHJlbjogJ2NoaWxkcmVuJyxcbiAgaXNMZWFmOiAnaXNMZWFmJyxcbn1cblxubGV0IGlkID0gMFxuXG5jb25zdCBsb2FkID0gKG5vZGUsIHJlc29sdmUpID0+IHtcbiAgaWYgKG5vZGUuaXNMZWFmKSByZXR1cm4gcmVzb2x2ZShbXSlcblxuICBzZXRUaW1lb3V0KCgpID0+IHtcbiAgICByZXNvbHZlKFtcbiAgICAgIHtcbiAgICAgICAgdmFsdWU6ICsraWQsXG4gICAgICAgIGxhYmVsOiBgbGF6eSBsb2FkIG5vZGUke2lkfWAsXG4gICAgICB9LFxuICAgICAge1xuICAgICAgICB2YWx1ZTogKytpZCxcbiAgICAgICAgbGFiZWw6IGBsYXp5IGxvYWQgbm9kZSR7aWR9YCxcbiAgICAgICAgaXNMZWFmOiB0cnVlLFxuICAgICAgfSxcbiAgICBdKVxuICB9LCA0MDApXG59XG5cblxuY29uc3QgZmlsdGVyTm9kZU1ldGhvZCA9ICh2YWx1ZSwgZGF0YSkgPT4gZGF0YS5sYWJlbC5pbmNsdWRlcyh2YWx1ZSlcbjwvc2NyaXB0PlxuIiwiaW1wb3J0LW1hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge31cbn0iLCJ0c2NvbmZpZy5qc29uIjoie1xuICBcImNvbXBpbGVyT3B0aW9uc1wiOiB7XG4gICAgXCJ0YXJnZXRcIjogXCJFU05leHRcIixcbiAgICBcImpzeFwiOiBcInByZXNlcnZlXCIsXG4gICAgXCJtb2R1bGVcIjogXCJFU05leHRcIixcbiAgICBcIm1vZHVsZVJlc29sdXRpb25cIjogXCJCdW5kbGVyXCIsXG4gICAgXCJ0eXBlc1wiOiBbXCJlbGVtZW50LXBsdXMvZ2xvYmFsLmQudHNcIl0sXG4gICAgXCJhbGxvd0ltcG9ydGluZ1RzRXh0ZW5zaW9uc1wiOiB0cnVlLFxuICAgIFwiYWxsb3dKc1wiOiB0cnVlLFxuICAgIFwiY2hlY2tKc1wiOiB0cnVlXG4gIH0sXG4gIFwidnVlQ29tcGlsZXJPcHRpb25zXCI6IHtcbiAgICBcInRhcmdldFwiOiAzLjNcbiAgfVxufVxuIiwiX28iOnt9fQ==)

**Reply by WestFarmer** [reply]
No, this example have no remote filtering at all. but thanks for your help.

**Comment by WestFarmer** [comment]
when user click at a `tree-select`, it's in lazy mode at first, but when user start typing in it, lazy mode should be turned off.

maybe I can do this, by make `lazy` reactive.
